core/src/main/java/com/squarespace/template/plugins/platform/ContentFormatters.java

```diff
@@ -320,6 +323,82 @@ public void apply(Context ctx, Arguments args, Variables variables) throws CodeE
     }
   };
 
+  public static class WebsiteColorFormatter extends BaseFormatter {
+
+    public WebsiteColorFormatter() {
+      super("website-color", false);
+    }
+
+    @Override
+    public void apply(Context ctx, Arguments args, Variables variables) throws CodeExecuteException {
+      Variable var = variables.first();
+      JsonNode node = var.node();
+      boolean hasAlphaValue = false;
+
+      double hue = node.path("hue").asDouble();
```

Comment:
I'd make the error messages a bit more concise, given that the longer strings will increase the code size for the Typescript compiler.